### DISCUSSION TITLE
Recently, a self-built mail system using mailu was used to send spam.

### DISCUSSION THREAD

**Discussion by hudingwen** [Mailu/Mailu#3397] [User support]
Recently, a self-built mail system using mailu was used to send spam, and then the mail was rejected by the receiving system. But I didn't see any records of how the self-built mail system was used by hackers to send spam? How can I check the logs?

Version: 2024.06.11

**Comment by hudingwen** [comment]
I don't see any sending logs in the anti-spam

**Comment by hudingwen** [comment]
The content of the returned email is as follows, but I checked the system and there is no sending record.

[
Dear user xtmy@xxxx.com:
Sorry to inform you that your email (send to xxxx@163.com) has been returned. The details are as follows:
Your email was marked as spam by the system and could not be sent.
Solutions: The email content may contain advertisements, fraud, phishing, etc., or the account reputation is poor, or there are a lot of reports. NetEase firmly opposes and resists any use of NetEase mailbox to send or transmit spam. It is not recommended to send commercial advertisements, publicity, promotion and other types of emails sent without the recipient's request through email. If the intercepted email is not spam, it is recommended that you try to modify the email subject, content, attachments, etc. and send it again.
------------------------ Original Mail Information ------------------------
Time: 2024-09-02 21:26:00
Subject: Chih Kai
To: xxxx@163.com
Relpy Code:
rejected by system
]

**Comment by ghostwheel42** [comment]
Hi. Why do you think this e-mail was sent by you mailu instance?
Is the IP address of your instance visible in the message (but omitted)?
You can find how to see the logs in the manual: https://mailu.io/2.0/maintain.html
Alex
